### PR TITLE
ROS-320: remove adm.* taglets

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

**Comment by Katsuya-Tomioka** [comment]
I admit it's a breaking change if ever depended on this custom tag for some reason. But we can't support it any longer and benefit is dubious as far as I can tell.

**Review by dscorbett — CHANGES_REQUESTED** [review]
Is this change needed? I can build `master` fine without this change.

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ script: mvn verify javadoc:aggregate
 jdk:
 - oraclejdk8
 - openjdk8
+dist: trust
```
[the remark this answers is not in the document]
Comment (Katsuya-Tomioka) [reply .travis.yml]: thanks. fixed.

**Comment by Katsuya-Tomioka** [comment]
I don't think we can build in a clean environment. I don't see any green builds in travis.
https://travis-ci.org/github/basis-technology-corp/annotated-data-model/builds

me or travis can't build it. If we know what's causing this build issue, and fix it reliably we could keep it.

**Review by dscorbett — APPROVED** [review]

**Pull request merged**